ports/rp2/boards/WEACTSTUDIO/mpconfigboard.cmake

```diff
@@ -0,0 +1,43 @@
+# CMake file for WeAct Studio RP2040 boards
+
+# The WeAct Studio boards don't have official pico-sdk support so we define it
+# here in weactstudio.h.
+# See also: https://github.com/raspberrypi/pico-sdk/tree/master/src/boards/include/boards
+list(APPEND PICO_BOARD_HEADER_DIRS ${MICROPY_BOARD_DIR})
+
+# Freeze board.py
+set(MICROPY_FROZEN_MANIFEST ${MICROPY_BOARD_DIR}/manifest.py)
+
+# Provide different variants for the downloads page.
+set(BOARD_VARIANTS "flash_2mb flash_4mb flash_8mb flash_16mb")
+
+if("${BOARD_VARIANT}" STREQUAL "flash_2mb")
```

Comment:
Which one is the default variant? I think you'll end up building PICO?

Reply:
I chose `flash_16mb` as the default; it seems like it'd be the most popular.